rosetta/create-distribution.sh

```diff
@@ -45,27 +56,26 @@ done
 if [[ $# -ge 1 ]]; then
     echo "Un-recognized argument: $*"
     echo
-    usage
+    usage 1
 fi
 
 set -euox pipefail
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
-INSTALLED_T5X_DIR=${INSTALLED_T5X_DIR:-/opt/t5x}
+INSTALLED_DIR=${INSTALLED_DIR:-/opt/t5x}
 DISTRIBUTION_BASE_REF=${DISTRIBUTION_BASE_REF:-main}
-INTERNAL_T5X_SUBMODULE_DIR=$SCRIPT_DIR/t5x
-UPSTREAM_GIT_URL=https://github.com/google-research/t5x.git
+UPSTREAM_GIT_URL=${UPSTREAM_GIT_URL:-https://github.com/google-research/t5x.git}
 
-if [[ -z "$INSTALLED_T5X_DIR" ]]; then
-  echo "[ERROR]: Need to specify the env INSTALLED_T5X_DIR="
+if [[ -z "$INSTALLED_DIR" ]]; then
+  echo "[ERROR]: Need to specify the env INSTALLED_DIR="
```

Comment:
Since we have the `-d/--dir` option for specifying the installation dir, we could print the error message plus the usage guide here.